tests/bluemira/geometry/test_data/div_test_mfm.json

```diff
@@ -1 +0,0 @@
4728295, -6.616086413152516, -6.616086413152516, -6.616105863824487, -6.620981730055064, -6.627485555130988, -6.627485556062311, -6.633989383932203, -6.633989383466542, -6.6345386281609535, -6.669824014417827, -6.680876425933093, -6.680876429658383, -6.697092400863767, -6.713308374397457, -6.7295243460685015, -6.729524349793792, -6.748424214776605, -6.753680896013975, -6.768947501201183, -6.786027857568115, -6.821435131132603, -6.821435132995248, -6.8391387718729675, -6.856842407491058, -6.8568424098193645, -6.8745460491627455, -6.8898171707987785, -6.901613043155521, -6.924231737852097, -6.9297762820497155, -6.953417622484267, -6.956446619704366, -6.977082214783877, -7.0153442597948015, -7.034475279971957, -7.034475282765925, -7.072737333830446, -7.072737335693091, -7.073779585771263, -7.074356267694384, -7.094388057012111, -7.114959197118878, -7.114959198515862, -7.135561092291027, -7.135622586123645, -7.157708551269025, -7.160670662298799, -7.182513405103236, -7.2028823136352, -7.223251223564148, -7.223251226823777, -7.243620140012354, -7.2862071599811316, -7.289882440585643, -7.34083714755252, -7.3921084948815405, -7.456670397426933, -7.496561153326184, -7.523248496465385, -7.524899006821215, -7.502254460006952, -7.465155387297273, -7.397054408676922, -7.329202237073332, -7.2288490370847285, -7.140960890799761, -7.051499701105058, -7.010096779093146, -6.850688190665096, -6.827524489257485, -6.827524485066533, -6.719685612434346, -6.526396286208183, -6.52639628527686, -6.503232577815652, -6.503232577815652, -6.480068877805024, -6.480068873614073, -6.413700918667018, -6.3980807065963745, -6.355950031429529, -6.333959701936692, -6.3205975871533155, -6.305839375592768, -6.258946730289608, -6.258946726564318, -6.246557333972305, -6.236039890907705, -6.214413818903267, -6.214413818903267, -6.190877659711987, -6.190877655521035, -6.176851005293429, -6.169908673502505, -6.153458732645959, -6.129952391143888, -6.112467097584158, -6.10947066033259, -6.10052112583071, -6.10052112210542, -6.05489255907014, -6.032431241124868, -6.006110856775194, -5.992417314089835, -5.970814513973892, -5.970814512111247, -5.960382487159222, -5.952377453446388, -5.952377453446388, -5.932107432279736, -5.932107430417091, -5.9150836849585176, -5.913261932786554, -5.908507171086967, -5.908507169224322, -5.889630299527198, -5.88963029673323, -5.8707534284330904, -5.870753427967429, -5.863856318872422, -5.858658609446138, -5.842298134230077, -5.842298134230077, -5.824656302575022, -5.824656302109361, -5.8126614657230675, -5.804456404410303, -5.791096241679043, -5.774437479209155, -5.774437476880848, -5.537956531625241, -5.325361884664744, -5.332005923148245, -5.344397844746709, -5.3565479423850775, -5.368418048135936, -5.3800504435785115, -5.3913968205451965, -5.402500879019499, -5.413324547465891, -5.423896882217377, -5.434193668887019, -5.444232070352882, -5.453998758923262, -5.463502215221524, -5.4727365602739155, -5.481705261860043, -5.478971095290035, -5.491773025598377]}
```

Comment:
I guess this test no longer works with the face being fixed if invalid?

Reply:
Yeah this Loop is just bad (it has overlapping lines) so a valid face can't be made from it.